rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -1199,32 +1255,20 @@ def get_erc721_contract_info(self, address: ChecksumEvmAddress) -> dict[str, Any
         if cache is not None:
             return cache
 
-        properties = ('symbol', 'name')
-        info: dict[str, Any] = {}
+        output = self._query_token_contract(token_abi_str='ERC721_TOKEN', properties=ERC721_PROPERTIES, address=address)  # noqa: E501
 
-        abi = self.contracts.abi('ERC721_TOKEN')
-        contract = EvmContract(address=address, abi=abi, deployed_block=0)
-        try:
-            # Output contains call status and result
-            output = self.multicall_2(
-                require_success=False,
-                calls=[(address, contract.encode(method_name=prop)) for prop in properties],
-            )
-        except RemoteError:
-            # If something happens in the connection the output should have
-            # the same length as the tuple of properties
-            output = [(False, b'')] * len(properties)
         try:
             decoded = self._process_contract_info(
                 output=output,
-                properties=properties,
-                contract=contract,
+                properties=ERC721_PROPERTIES,
+                contract=EvmContract(address=address, abi=self.contracts.abi('ERC721_TOKEN'), deployed_block=0),  # noqa: E501
```

Comment:
wouldn't it make sense to store 

- self.contracts.abi('ERC20_TOKEN')
- self.contracts.abi('ERC721_TOKEN')

in memory to avoid multiple queries to the db + json load?

Reply:
This is the only thing I have not yet touched. I will push to other commit or PR. I would not store it in `self`. As there is as many `selves` as there is EVM chains. And this is universal across all of them. So need to rethink some stuff here.